### ISSUE TITLE
When e.target is bigger than viewport...

### ISSUE THREAD

**Issue by panksi** [vtimofeev/vue-check-view#9]
So I have an element target which is bigger than viewport size,
and 
viewHandler(e) fires exit from element before it actually leaves the viewport
here is the printscreen of console.log
![image](https://user-images.githubusercontent.com/16855132/62535344-75355c00-b84b-11e9-96ef-9461eeb9787b.png)

I think this is because element is bigger and then something trippy happens to percent in view
![image](https://user-images.githubusercontent.com/16855132/62535471-c9404080-b84b-11e9-9b90-89cd76eb44bb.png)

Its not a big issue but I think its worth fixing and making this plugin SUPERAWESOME!
Anyway thanks for even making this, it really saved me the time

**Comment by Suv4o** [comment]
I am facing the same issue when an element is bigger than the viewport. It also happens with "e.percentTop". Basically it dose not indicates when you are in the middle of an element. It would be nice if you can implement this little feature as well.

<img width="369" alt="Screen Shot 2020-01-01 at 13 59 28" src="https://user-images.githubusercontent.com/56303591/71637740-29be9700-2c9f-11ea-9661-fe29d2dd000a.png">

Anyway, thank you very much for the great package. Good work!

**Comment by simonhrogers** [comment]
Yeah this is currently causing me woes too – I really need it to work for longer elements, as my use-case is tracking scroll position down a series of text elements for a navigation which highlights those currently on the screen.

**Comment by simonhrogers** [comment]
For what it’s worth, this workaround worked for me:

```
    viewHandler(e, section) {
      if (e.percentTop >= 0 && e.percentTop <= 1) {
        this.visibleSections.push(section);
      } else {
        this.visibleSections = this.visibleSections.filter(item => item._key !== section._key);
      }
    }
```
